src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -560,9 +639,13 @@ private static Collection<ScriptBindingProvider> LoadBindingProviders(ScriptHost
                 typeof(NotificationHubScriptBindingProvider),
                 typeof(SendGridScriptBindingProvider),
                 typeof(TwilioScriptBindingProvider),
-                typeof(BotFrameworkScriptBindingProvider)
-            };
+                typeof(BotFrameworkScriptBindingProvider),
```

Comment:
As part of this review, I think we should remove all the built in ScriptBindingProviders and replace them with the single GeneralScriptBindingProvider. That's the goal of this work, and w/o doing this, it isn't clear that the new stuff handles all the scenarios.